src/fsd/4-entities/unit/characters-power.service.ts

```
@@ -131,34 +156,65 @@ export class CharactersPowerService {
+                    return 1.8;
+                case RarityStars.RedFourStars:
+                    return 1.9;
+                case RarityStars.RedFiveStars:
+                    return 2.0;
+                case RarityStars.BlueStar:
+                    return 2.1;
 
-            case RarityStars.None:
-            default:
-                return 1.0;
+                case RarityStars.None:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)